### PR TITLE
Makefile for windev

### PR DESCRIPTION
**Pull request by voisardf**
Added an new target to the Makefile for a local windows environment with paster

### CODE REVIEW

**Comment by voisardf** [comment]
I accidentaly added some files I need to remove first

**Comment by voisardf** [comment]
@kdeininger I removed the unecessary files, PR is ready for review now.
I do not know yet, why the CI fails

File: Makefile [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # Variable definitions
 # ********************
 
+# Environment and python
+VENV_BIN ?= .venv/Scripts/
```
Comment (kdeininger) [inline Makefile:6]: The CI fails with the following error:

```
make: .venv/Scripts/pip: Command not found
```

You need to use `VENV_BIN ?= .venv/bin/` here and create a new file, e.g. `windows.mk` with the follwing content:

```
VENV_BIN ?= .venv/Scripts/
include Makefile
```

Then use `make -f windows.mk serve-windev` to start the server.

**Review by kdeininger — CHANGES_REQUESTED** [review]

**Comment by voisardf** [comment]
@kdeininger : should be fixed now. Working on local dev

**Review by kdeininger — APPROVED** [review]

**Review by kdeininger — APPROVED** [review]

**Pull request merged**